### ISSUE TITLE
Error when applying

### ISSUE THREAD

**Issue by ssssamstrup** [FoxP/PimpMyDrive#1]
Windows 11:

When I've picked a drive and an icon to add to it and click apply, I get this error message:
![Skærmbillede 2024-11-11 142659](https://github.com/user-attachments/assets/742bc82c-b4d1-4c45-94c6-48b043548c1e)

It doesn't seem like file permissions is the culprit:
![Skærmbillede 2024-11-11 143111](https://github.com/user-attachments/assets/783d811e-36fc-495a-8a76-6981a17ccf63)

Permissions for user and Administrators are the same as in the screenshot for SYSTEM

**Comment by FoxP** [comment]
Thanks for the bug report. 👍 
Do you have the same issue when starting the app with admin rights ?

**Comment by ssssamstrup** [comment]
That fixed it. It would be great if you can change the error message to say that the program needs to be run as administrator to work? The current error message didn't reflect that could be the issue to me.

**Comment by FoxP** [comment]
Yes, I definitely need to handle this error with a better error message. 👍

**Comment by ssssamstrup** [comment]
Thank you!
